### ISSUE TITLE
Use tagged template literals in connect

### ISSUE THREAD

**Issue by maxfrigge** [cerebral/cerebral#563]
Just came to my mind and wanted to see what others think about this.

```js
connect({
  foo: state`foo.path`,
  computed: computed`foo.augment`,
  signal: signal`foo.signal`
}, (connectProps, ownProps) => ({
  foo: connectProps.foo,
  ownFoo: ownProps.foo
}), (props) => {
  ... 
})
```

**Comment by gaspard** [comment]
The advantage of this is that signals are not in a separate object but it requires more imports "state", "signal" at least and doesn't add much. It also makes the dev experience harder to get started (have to learn about template tags to use components).

**Comment by christianalfoni** [comment]
https://github.com/cerebral/cerebral/issues/568

**Comment by maxfrigge** [comment]
obsolete since it's included in #568
